Dockerfile

```diff
@@ -26,6 +26,8 @@ COPY packages/transition-backend packages/transition-backend
 COPY packages/transition-frontend packages/transition-frontend
 RUN yarn install
 
+# Setup the example as a default configuration for the image
+COPY .env.example /app/.env
```

Comment:
@greenscientist  Can this be a security issue? The local .env may contain sensible information and now has precedence over the .env.example.

But currently, the .env.example does not work, as the PROJECT_CONFIG points to nowhere. Maybe the .env.example could just have the internal docker path? `/app/examples/config.js` instead of `/absolute/path/to/config.js`?

But then how does one run docker with a custom .env file?

Reply:
You have to add the PROJECT_CONFIG to the environnement of your docker execution and not rely on what's in the docker file.